src/Hazelcast.Net.Examples/SoakTests/Soak1.cs

```diff
@@ -0,0 +1,191 @@
+﻿// Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Hazelcast.DistributedObjects;
+using Hazelcast.Examples.Models;
+using Hazelcast.Query;
+using Microsoft.Extensions.Logging;
+
+namespace Hazelcast.Examples.SoakTests
+{
+    // ./hz.ps1 run-example ~Soak1 --%
```

Comment:
Is  `--%` needed if commands ends with it?

Reply:
as in, if we run without arguments at all? no, not needed - that's a PowerShell thing, PS is super-annoying with arguments